lib/v2/navigation/navigation_service.dart

```diff
@@ -129,7 +129,7 @@ class NavigationService {
     Routes.invites: (_) => Invites(),
     Routes.createInvite: (_) => const InviteScreen(),
     Routes.vote: (_) => const VoteScreen(),
-    Routes.proposalDetailsPage: (args) => ProposalDetailsPage(proposal: args),
+    Routes.proposalDetails: (args) => ProposalDetailsScreen(args),
```

Comment:
To be honest I'm not really sure which pattern is better without going through the parameter's constructor or through navigation.